presentation/chapters/fr-FR/overview.chapter

```diff
@@ -25,7 +25,7 @@
 ## Développement
 
 -   Toujours avec un grand projet (ex: Servo)
--   Une adoption très tôt des releases régulière, depecrations et RFC processus
+-   Une adoption très tôt avec des releases régulières, et des obsolescences et RFC processus
```

Comment:
```suggestion
-   Une adoption en amont du projet de release régulière (tous les deux mois), d'un système de deprecation et d'un système développement basé sur des RFC.
```

Reply:
Nowadays, the release process is every six weeks, I wouldn't tell people about the _real_ old one.